plugins/out_es/es_conf.c

```diff
@@ -153,8 +157,32 @@ struct flb_elasticsearch *flb_es_conf_create(struct flb_output_instance *ins,
             flb_es_conf_destroy(ctx);
             return NULL;
         }
+        flb_plg_debug(ctx->ins, "extracted cloud_host: '%s'", cloud_host);
+
+        flb_plg_debug(ctx->ins,
+                      "check whether extracted cloud_host '%s' contains a port",
+                      cloud_host);
+        cloud_host_char = strtok_r(cloud_host, ":", &saveptr);
```

Comment:
Is `cloud_host_char` not used? Does we need to update `ins->host.name` ?
If we use `cloud_host_char`, we need to check it is NULL or not.